### PR TITLE
Create io.github.thiefmd.themegenerator manifest

### PR DESCRIPTION
**Pull request by kmwallio**
Tool for generating GtkSourceView Style Schemes and Ulysses Themes.

### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: io.github.thiefmd.themegenerator.json [hunk 1]
```diff
@@ -0,0 +1,52 @@
+{
+    "app-id":"io.github.thiefmd.themegenerator",
+    "runtime":"org.gnome.Platform",
+    "runtime-version":"3.38",
```
Comment (hfiguiere) [inline io.github.thiefmd.themegenerator.json:4]: Runtime 40 is out now, it is preferable to use that.

File: io.github.thiefmd.themegenerator.json [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    "sdk":"org.gnome.Sdk",
+    "command":"io.github.thiefmd.themegenerator",
+    "finish-args":[
+        "--share=ipc",
+        "--device=dri",
+        "--socket=fallback-x11",
+        "--socket=x11"
```
Comment (hfiguiere) [inline io.github.thiefmd.themegenerator.json:11]: It's either `x11` alone or `fallback-x11` with `wayland`. And suspect the latter is probably what you want.

**Comment by hfiguiere** [comment]
bot, build io.github.thiefmd.themegenerator

**Comment by hfiguiere** [comment]
Build is failing because the screenshot points to a URL that's not a picture (it's not the raw file on github)

**Comment by kmwallio** [comment]
Screen shots should be fixed.

Thank you for looking at this!

**Comment by hfiguiere** [comment]
bot, build io.github.thiefmd.themegenerator

**Comment by kmwallio** [comment]
> Build [43301 successful](https://flathub.org/builds/#/builders/32/builds/43301)
> To test this build, install it from the testing repository:
> 
> ```
> flatpak install --user https://dl.flathub.org/build-repo/41656/io.github.thiefmd.themegenerator.flatpakref
> ```

Build works as expected for me.

**Pull request closed without merging**